src/examples/tags/TagsClose.tsx

```diff
@@ -9,20 +9,36 @@ import React from 'react';
 import { Tag } from '@zendeskgarden/react-tags';
 import { KEY_CODES } from '@zendeskgarden/container-utilities';
 import { Row, Col } from '@zendeskgarden/react-grid';
+import { Tooltip } from '@zendeskgarden/react-tooltips';
 
-const handleKeyDown = (e: React.KeyboardEvent<any>) => {
+const handleTagKeyDown = (e: React.KeyboardEvent<any>) => {
   if (e.keyCode === KEY_CODES.DELETE || e.keyCode === KEY_CODES.BACKSPACE) {
     e.preventDefault();
     alert('Tag dismissed via keyboard');
   }
 };
 
+const handleCloseKeyDown = (e: React.KeyboardEvent<any>) => {
+  const KEYS = [KEY_CODES.SPACE, KEY_CODES.ENTER, KEY_CODES.DELETE, KEY_CODES.BACKSPACE];
+
+  if (KEYS.includes(e.keyCode)) {
+    e.preventDefault();
+    alert('Tag dismissed via keyboard');
+  }
+};
+
 const Example = () => (
   <Row>
     <Col textAlign="center">
-      <Tag tabIndex={0} onKeyDown={handleKeyDown}>
+      <Tag tabIndex={0} aria-label="Algae, press delete to remove" onKeyDown={handleTagKeyDown}>
         <span>Algae</span>
-        <Tag.Close onClick={() => alert('Tag dismissed via mouse')} />
+        <Tooltip content="Remove tag">
```

Comment:
Yikes. We _definitely_ cannot have a double nav on either `<Tag>` or `<File>`. This would add serious issues to product. I wonder if that brings us full circle to why we had these components structured as they originally were? 😬.

Reply:
Perhaps my previous comment was super clear - there's only a double tab on the website, in this PR specifically, because of the changes in the PR itself (adding Tooltip to File.Close, File.Delete, and Tag.Close without the latest changes). That double tab will go away once we uptake the next react-components version on the website and I rebase this branch. Currently there isn't a double tab from File or Tag components.

Reply:
That's a relief. I'll re-review once `react-components` is released and updated here.

Reply:
Updated. I can split the dep updates to a separate PR if desired.